README.md

```diff
@@ -1,26 +1,163 @@
 # LifeHash: Beautiful Visual Hashes
 
-This page is a demonstration of running the [LifeHash](https://github.com/BlockchainCommons/bc-lifehash) visual hash algorithm in the web browser. It uses the reference C++ implementation compiled to [WebAssembly](https://webassembly.org/). The page itself can be accessed [here](http://lifehash.info).
+* <img src="https://github.com/BlockchainCommons/crypto-commons/blob/master/images/logos/crypto-commons-super-simple.png" width=16 valign="bottom">&nbsp;&nbsp; ***part of the [crypto commons](https://github.com/BlockchainCommons/crypto-commons/blob/master/README.md) technology family***
 
-## Displaying a Specific LifeHash
+### _by [Wolf McNally](https://www.github.com/wolfmcnally) and [Christopher Allen](https://www.github.com/ChristopherA)_
 
-You can access this page's URL with a query parameter to populate the string input field with a specific value. Example:
+<a href="https://lifehash.info"><img src="Nexus-Images/lifehash-info.png" align="right" width=400></a>
+This is a central repo for LifeHash, a method of visual hash visualization that creates beautiful, deterministic icons.
```

Comment:
"Visual hash visualization" is redundant. It should either be "visual hashing" or "hash visualization."

Reply:
Replaced with "hash visualization"